runtime/v2/bundle.go

```diff
@@ -128,8 +129,10 @@ type Bundle struct {
 func (b *Bundle) Delete() error {
 	work, werr := os.Readlink(filepath.Join(b.Path, "work"))
 	rootfs := filepath.Join(b.Path, "rootfs")
-	if err := mount.UnmountAll(rootfs, 0); err != nil {
-		return fmt.Errorf("unmount rootfs %s: %w", rootfs, err)
+	if runtime.GOOS != "darwin" {
+		if err := mount.UnmountAll(rootfs, 0); err != nil {
+			return fmt.Errorf("unmount rootfs %s: %w", rootfs, err)
+		}
```

Comment:
Is there any possibility that a future version of Darwin may support nullfs (bind-mount) ?
If yes, the runtime has still try to unmount the rootfs (and ignore the error)

Reply:
Support of mounts on MacOS is fairly limited today and I'm not aware of any plans to support new features. I'm happy to change this once mounts are more usable on this platform.

Reply:
> Is there any possibility that a future version of Darwin may support nullfs (bind-mount) ?

See https://github.com/containerd/containerd/pull/8789 (and yes, it reverts this change).